main/constants/constants.ts

```diff
@@ -1,6 +1,6 @@
-export const DI_ROOT_INJECTOR_KEY = 'morganstanley.com.root.injector';
-export const DI_GLOBAL_STATE_STORE = 'morganstanley.com.state.store';
-export const GLOBAL_INSTANCE_MAP = 'morganstanley.com.instance.map';
-export const GLOBAL_REGISTRATION_MAP = 'morganstanley.com.registrations.map';
-export const GLOBAL_CONFIGURATION = 'morganstanley.com.configuration';
+export const DI_ROOT_INJECTOR_KEY = 'needle.morganstanley.com.root.injector';
+export const DI_GLOBAL_STATE_STORE = 'needle.morganstanley.com.state.store';
+export const GLOBAL_INSTANCE_MAP = 'needle.morganstanley.com.instance.map';
+export const GLOBAL_REGISTRATION_MAP = 'needle.morganstanley.com.registrations.map';
```

Comment:
yeah, I would have done `morganstanley.com.needle.registrations.map` so it goes from organisation, project, specifics but it doesn't really matter.